docs/en/integrations/index.asciidoc

```diff
@@ -0,0 +1,48 @@
+include::{docs-root}/shared/versions/stack/{source_branch}.asciidoc[]
+include::{docs-root}/shared/attributes.asciidoc[]
+
+:doctype: book
+:forum: https://discuss.elastic.co/
+
+:package-spec-repo: https://github.com/elastic/package-spec
+
+// Access package spec files with {package-spec-root}
+:package-spec-version: 1
+:package-spec: {package-spec-root}/versions/{package-spec-version}
+
+// repos
+:integrations-repo-link: https://github.com/elastic/integrations
+:package-registry-repo-link: https://github.com/elastic/package-registry
+:package-spec-repo-link: https://github.com/elastic/package-spec
+:package-storage-repo-link: https://github.com/elastic/package-storage
+
+// Additional information that hasn't been copied over yet
+// https://github.com/elastic/integrations/blob/master/docs/definitions.md
+// https://github.com/elastic/integrations/blob/master/docs/definitions.md
+
+= Integrations Developer Guide
+
+include::what-is-an-integration.asciidoc[leveloffset=+1]
+
+include::quick-start.asciidoc[leveloffset=+1]
+
+include::build-integration.asciidoc[leveloffset=+1]
+
+include::testing.asciidoc[leveloffset=+1]
+
+include::publish-integration.asciidoc[leveloffset=+1]
+
+include::elastic-package.asciidoc[leveloffset=+1]
+
+// include::dashboards.asciidoc[leveloffset=+1]
+
+include::package-spec.asciidoc[leveloffset=+1]
+
+include::pull-request.asciidoc[leveloffset=+1]
+
+// include::integration-example.asciidoc[leveloffset=+1]
+
+== Community integrations
+
+Should we have a community integrations page?
```

Comment:
Currently we don't have any and most of integrations are stored in the elastic/integrations repository. Maybe let's hide this subpage until we have at least 2 community integrations?